frame/society/src/lib.rs

```diff
@@ -1565,7 +1565,7 @@ impl<T: Config<I>, I: Instance> Module<T, I> {
 				// Start a new defender rotation
 				let phrase = b"society_challenge";
 				// we'll need a random seed here.
-				let seed = T::Randomness::random(phrase);
+				let (seed, _) = T::Randomness::random(phrase);
```

Comment:
Should there be a TODO here to update the logic to use this new block number?

Reply:
yup